src/rest_framework_dso/response.py

```diff
@@ -76,7 +76,20 @@ def _read_rendered_content(self):
                 f"No stream generated by {self.accepted_renderer.__class__.__name__}"
             )
 
-        yield from stream
+        try:
+            yield from stream
+        except Exception as e:
+            yield self.render_exception(e)
```

Comment:
Not sure if this provides enough info. For the end user nor for us developers (when `settings.DEBUG == True`).
You should probably use [sys.exc_info()](https://docs.python.org/3.8/library/sys.html#sys.exc_info) here. Something along the lines of (untested):

```python
yield self.render_exception(*sys.exc_info())  # render_exception()'s signature should change for this to work)
```
`render_exception()` should be changed to something like (untested):
```python
def render_exception(self, type: Type[Exception], value: Exception, tb: Any) -> str:
    """Inform clients that the stream was interrupted by an exception."""
    if settings.DEBUG:
        return '"\n\n" {stack_trace}'.format(
            stacktrace="".join(traceback.format_exception(type, value, tb)))
    else:
        return '"\n\nRendering aborted due to: {exc_msg}'.format(
            exc_msg="".join(traceback.format_exception_only(type, value)))
```

Reply:
This method only exists to infor the client that the stream processing was interrupted with an exception. Purposefully, not much information is given, so avoid informing clients with internal server details. The actual exception is still raised and logged server-side.